README.md

```diff
@@ -55,13 +55,63 @@ The server can currently run in two modes: `standalone` and `forwarder`.  It is
 In `standalone` mode, raw metrics are processed and aggregated as normal, and aggregated data is submitted to
 configured backends (see below)
 
+This configuration mode allows the following configuration options:
+- `expiry-interval`: interval before metrics are expired, see `Metric expiry and persistence` section.  Defaults
+   to `5m`.  0 to disable, -1 for immediate.
+- `expiry-interval-counter`: interval before counters are expired, defaults to the value of `expiry-interval`.
+- `expiry-interval-gauge`: interval before gauges are expired, defaults to the value of `expiry-interval`.
+- `expiry-interval-set`: interval before sets are expired, defaults to the value of `expiry-interval`.
+- `expiry-interval-timer`: interval before timers are expired, defaults to the value of `expiry-interval`.
+- `flush-aligned`: whether or not the flush should be aligned.  Setting this will flush at an exact time interval.  With
+  a 10 second flush-interval, if the service happens to be started at 12:47:13, then flushing will occur at 12:47:20,
+  12:47:30, etc, rather than 12:47:23, 12:47:33, etc.  This removes query time ambiguity in a multi-server environment.
+  Defaults to `false`.
+- `flush-interval`: duration for how long to batch metrics before flushing. Should be an order of magnitude less than
+  the upstream flush interval. Defaults to `1s`.
+- `flush-offset`: offset for flush interval when flush alignment is enabled.  For example, with an offset of 7s and an
+  interval of 10s, it will flush at 12:47:10+7 = 12:47:17, etc.
+- `ignore-host`: indicates whether or not an explicit `host` field will be added to all incoming metrics and events.
+  Defaults to `false`
+- `max-readers`: the number of UDP receivers to run.  Defaults to 8 or the number of logical cores, whichever is less.
+- `max-parsers`: the number of workers available to parse metrics.  Defaults to the number of logical cores.
+- `max-workers`: the number of aggregators to process metrics.  Defaults to the number of logical cores.
+- `max-queue-size`: the size of the buffers between parsers and workers.  Defaults to `10000`, monitored via
+  `channel.*` metric, with `dispatch_aggregator_batch` and `dispatch_aggregator_map` channels.
+- `max-concurrent-events`: the maximum number of concurrent events to be dispatching.  Defaults to `1024`, monitored
+  via `channel.*` metric, with `backend_events_sem` channel.
+- `estimated-tags`: provides a hint to the system as to how many tags are expected to be seen on any particular metric,
+  so that memory can be pre-allocated and reducing churn.  Defaults to `4`.  Note: this is only a hint, and it is safe
+  to send more.
+- `log-raw-metric`: logs raw metrics received from the network.  Defaults to `false`.
```

Comment:
Should this be deprecated in favor of the stdout backend?

Reply:
Nah, that's post-aggregation, this is specifically for pre-aggregation, I believe it logs the metric the client send after parsing.  I believe @beanliu needed it for something.